tests/suffixarray_test.py

```diff
@@ -0,0 +1,448 @@
+import numpy as np
```

Comment:
@hokiegeek2 will know for sure, but I'm pretty sure this needs to be added to `pytest.ini` to run as part of `make test` and CI testing.

Reply:
The reason lies in two aspects: (1) the existing research in suffix array show that different algorithms may have very different performance for different inputs (for examples, the average lcp length), so no one algorithm is the best for all. 2 we want to provide automatical optimization mechanism at run-time. If we know the input features, we can select the suitable algorithm to achieve good practical performance.   At the same time, we want to evaluate the effect of two mechanisms. One mechanism is integrating the existing codes, and the other is developing it by ourselves in Chapel. We want to do some comparison on how the two mechanism can work together in Arkouda.

Reply:
About the third party code, I agree that this will often introduce some problem. Another consideration is that Arkouda hopes to integrate existing parallel library (from the presentation), so I do this just want to check the feasibility,  performance and possible problem (can only use one locale now).

Reply:
> @hokiegeek2 will know for sure, but I'm pretty sure this needs to be added to `pytest.ini` to run as part of `make test` and CI testing.

Correct